apps/wallet-mobile/src/HW/LedgerConnect/LedgerConnect.tsx

```diff
@@ -305,7 +305,11 @@ class _LedgerConnect extends React.Component<Props, State> {
   }
 }
 
-export const LedgerConnect = injectIntl(_LedgerConnect)
+export const LedgerConnect = (props: Omit<Props, 'intl'>) => {
```

Comment:
even with `// @ts-ignore` it was complaining - `react-intl` uses `@types/react` still is not bringing the right types of the latest version of react 18.2